### PR TITLE
[SDFAB-989] Setup basic CI

### PR DESCRIPTION
**Pull request by EmanueleGallone**
PR's aim is to setup basic CI for `pfcpsim`

~~Blocked by #2~~ 

TODO:
- [x] Use GH-actions to run linter
- [x] Fix go linter.
- [x] Setup basic build of `pfcpsim-client`

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.16
+
+      - name: Check out repository
+        uses: actions/checkout@v1
+
+      - name: Build
+        run: go build cmd/pfcpsim-client/main.go
```
Comment (osinstom) [inline .github/workflows/main.yml:20]: shouldn't we use docker build?

File: Makefile [hunk 2]
```diff
@@ -17,3 +17,6 @@ build-pfcpsim-client:
 	DOCKER_BUILDKIT=$(DOCKER_BUILDKIT) docker build -f Dockerfile . \
 	--target ${DOCKER_TARGET} \
 	--tag ${DOCKER_REGISTRY}${DOCKER_REPOSITORY}${DOCKER_TARGET}:${DOCKER_TAG} \
+
+golint:
+	@docker run --rm -v $(CURDIR):/app -w /app/pkg/pfcpsim golangci/golangci-lint:latest golangci-lint run -v --config /app/.golangci.yml
```
Comment (osinstom) [inline Makefile:22]: no newline at EOF

File: .github/workflows/main.yml (hunk 1 above)
Comment (EmanueleGallone) [reply .github/workflows/main.yml]: yes. Now it uses docker build.
CI however does not release during test step. Need to investigate

File: utils/errors/errors.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package errors
```
Comment (osinstom) [inline utils/errors/errors.go:1]: it's fine to put `errors.go` in `pkg/pfcpsim`

**Review by osinstom — CHANGES_REQUESTED** [review]

**Review by osinstom — APPROVED** [review]

**Pull request merged**